node/shard/src/lib.rs

```diff
@@ -307,11 +307,11 @@ mod tests {
 
     use super::*;
 
-    fn get_test_client() -> (ShardClient, Arc<InMemorySigner>) {
-        let (chain_spec, signer) = generate_test_chain_spec();
+    fn get_test_client() -> (ShardClient, Vec<Arc<InMemorySigner>>) {
```

Comment:
This change seems to be unrelated to the catchup. Should we mention it in the PR description separately?